web3/queries/votes/getEncryptedVotes.ts

```diff
@@ -1,25 +1,31 @@
-import { VotingV2Ethers } from "@uma/contracts-frontend";
+import { VotingEthers, VotingV2Ethers } from "@uma/contracts-frontend";
 import { decodeHexString, makeUniqueKeyForVote } from "helpers";
 import { EncryptedVotesByKeyT } from "types";
 
 export async function getEncryptedVotes(
   votingContract: VotingV2Ethers,
-  address: string,
-  roundId: number
+  votingV1Contract: VotingEthers,
+  address: string
 ) {
-  const filter = votingContract.filters.EncryptedVote(
-    address,
-    roundId,
-    null,
-    null,
-    null,
-    null
-  );
-  const result = await votingContract.queryFilter(filter);
+  const v1Filter = votingV1Contract.filters.EncryptedVote(address);
+  const v1Result = await votingV1Contract.queryFilter(v1Filter);
+
+  const v2Filter = votingContract.filters.EncryptedVote(address);
+  const v2Result = await votingContract.queryFilter(v2Filter);
+
+  const v1EventData = v1Result?.map(({ args }) => args);
+  const v2EventData = v2Result?.map(({ args }) => args);
 
-  const eventData = result?.map(({ args }) => args);
   const encryptedVotes: EncryptedVotesByKeyT = {};
-  eventData?.forEach(({ encryptedVote, identifier, time, ancillaryData }) => {
+
+  v1EventData?.forEach(({ encryptedVote, identifier, time, ancillaryData }) => {
+    const decodedIdentifier = decodeHexString(identifier);
+    encryptedVotes[
+      makeUniqueKeyForVote(decodedIdentifier, time, ancillaryData)
+    ] = encryptedVote;
+  });
```

Comment:
my fear with this is that v1 vote data is somehow different of off from v2 data, which may require extra knowledge that its data coming from v1, but i guess we can cross that bridge if its a problem

Reply:
yeah I agree, we'll have to keep an eye out. still, this is just for historic data so its not the end of the world if we encounter a bug here